src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(numbers) {
+    if (numbers === undefined) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    }
+    sum += numbers;
```

Comment:
It's a good practice to ensure that the input is a number before adding it to 'sum'. You might want to add a check here to ensure 'numbers' is indeed a number to avoid potential bugs.